### ISSUE TITLE
Create a ✨TGIF team✨ with regular curation contributors

### ISSUE THREAD

**Issue by renrutnnej** [hoodiehq/editorial#96]
[TGIF](https://github.com/hoodiehq/editorial/wiki/How-to-create-TGIF) is one of the main functions of **Hoodie’s editorial team**. 

Because “code things” only make up one portion of our project and community, since the beginning Hoodie has put together a weekly curated list of interesting reads, videos, and thoughts that apply to areas like **open source**, **design**, **business**, and random **things we love**. 

You can see examples of TGIF below:

+ [Email heaven redefines education, TGIF 97](http://hood.ie/blog/email-heaven-redefines-education-tgif-97.html)
+ [Builders celebrate mediocre men apocalypse survival, TGIF 95](http://hood.ie/blog/builders-celebrate-mediocre-men-apocalypse-survival-tgif-95.html)
+ [Async emojis go Offline Camp, TGIF 90](http://hood.ie/blog/async-emojis-go-offline-camp-tgif-90.html)
+ [Upgrading giant golden robots beyond the boring, TGIF 72](http://hood.ie/blog/golden-robots-tgif-72.html)

Currently @skeskali is our TGIF curator and she does an awesome job. Moving forward, we would like to change it up and see if we can get a rotating roster of curators to be combing the interwebs for interesting and relevant Hoodie community-esque type content. 

The time commitment is low (I’d like @skeskali to give a time estimate below, if possible), since TGIF gets posted just once a week on Fridays, however, it might take a decent chunk of time getting up to speed if you’re unfamiliar with Hoodie, GitHub, open source, etc. 

That being said, **you do not have to be a programmer**, or **have open source experience** to be a curator. You just have to have a good eye for a good read, sprinkled with some enthusiasm and willingness to try new things. 

Ideally we’d have 😍 4-6 regular contributors, who @skeskali and myself would check in with to confirm who is covering when, etc. and also provide with editing and support. 

If this sounds like something you, or someone you know would be interested in trying, check out the [How to create TGIF guide](https://github.com/hoodiehq/editorial/wiki/How-to-create-TGIF) in the editorial wiki, and then comment below, or tag below. 

This is just an idea so we’re **very open** to how the TGIF team would be structured, as well as how TGIF is currently put together. 👉🏻 👉🏻 👉🏻Ideas and suggestions are very welcome! ☺️

**Comment by Charlotteis** [comment]
LGTM!

**Comment by skeskali** [comment]
I wouldn't say the time commitment is low. On average, I spend between 4-6 hours a week putting it together. I read every article I link to just to make sure it meets with Hoodie's standards, and to make sure it's worthy of including. Other people may not need this much time, but Github and publishing are the easiest parts of the process for me.

**Comment by machikoyasuda** [comment]
I am interested in recruiting friends in LA for this and I can pair with them in person if need be!

**Comment by renrutnnej** [comment]
That's great feedback @skeskali! I'll adjust the issue to say the time commitment is moderate, around 5 hours per week, is that fair?

**Comment by gr2m** [comment]
Yeah I agree with @skeskali, the one time I did TGIF myself, I spent a few hours on it, because I don’t spend so much time reading articles usually, so I couldn’t just pick some that I already knew, instead went trough all the articles myself that I thought sounded good.

I’d also suggest to link to http://hood.ie/about/, just to get everyone understand what our values are, what kind of things we are interested in?

Other than that, LGMT 💯 Thanks for pushing this forward 👏 

I think we also discussed that we can split up the categories of the TGIF, like Business, Open Source, Design, and have different people working on it, to split up the work? But maybe that’s the next step?

**Comment by skeskali** [comment]
I'd really love to have someone else handle the Open Source section, as it's the one I find most challenging (I don't have enough knowledge/experience in the area to know whether the links I'm including are any good).

**Comment by machikoyasuda** [comment]
@skeskali - @ahealthydesign and I can pair edit/write/emoji/markdown the Open Source section. I have been collecting blog posts, GitHub projects, newspaper articles, academic papers about open source and I need help organizing and posting them all.

ps. hello! 🚲 ❤️

**Comment by machikoyasuda** [comment]
Adding @elrandomhero to this thread -- @ahealthydesign (Rails Girls LA 2013/2014 alum) and @elrandomhero are advocates, writers, multilingual social media sensations that have extensive experience in open-source apps like @civicrm, communities like @nationbuilder, @wordpress and good use of emojis & gifs. We all worked together teaching bike safety in LA. Excited to mentor them through github.com so they can start contributing <3
